keyboards/ergo42/keymaps/biacco-winjp/rules.mk

```diff
@@ -0,0 +1,3 @@
+ifndef QUANTUM_DIR
```

Comment:
This block isn't actually needed.

It's a holdover from the old build system, and is no longer needed.

Reply:
Thank you for your comment. I'll remove this file.

Reply:
I've changed in this commit 62da35313da0429cd180f674cad9523664bef05b
`rules.mk` file was removed.